create_box/main.py

```diff
@@ -1,9 +1,28 @@
 """This is the entry point of the program."""
 
 
+
 def create_box(height, width, character):
-    pass
+    box= ''
+    if height < 1 or width < 1:
+        return 'Error'
+    for x in range(height):
+        box+= character*width+'\n'
```

Comment:
+1 to solution. Check PEP8 for standards about formatting. It's a good practice to include spaces between operators: `box += character * width + '\n'`.